book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Hi great job so far. As part of the requirements, you are supposed to create a has-many relationship side of Book and Rental (a book has many rentals). So you are supposed to have a method to add rentals in the book class